aten/src/ATen/cpu/vml.h

```diff
@@ -35,21 +35,8 @@
 #include <mkl.h>
 #endif
 
-// [Note SSE-AVX transitions]
-// There is a bug in Glibc2.23
-// https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1663280. Calling zeroall
-// when using AVX/AVX2 code resolves this.
-#if defined(CPU_CAPABILITY_AVX) && defined(__GLIBC__) && __GLIBC_MINOR__ == 23
-#define DL_RUNTIME_BUG(op, type_)                              \
-  using value_t = typename c10::scalar_value_type<type_>::type;\
-  volatile value_t x = (value_t)(1);                           \
-  x = std::op(x);                                              \
-  _mm256_zeroall();
-#define DL_RUNTIME_BUG_BFLOAT16() _mm256_zeroall();
-#else
```

Comment:
Can you please explain why this could be deleted?

Reply:
It's only being used for  `CPU_CAPABILITY_AVX`, which would be removed in this PR. I changed the title to reflect this. Thanks!